minutes/minutes-2017-03-08.md

```diff
@@ -0,0 +1,142 @@
+
+**Secretary** 
+- Candidate: Rayna Harris
+- Qualifications/Statement: Interested. Good at organizing, taking notes, adding minutes, writing blogs, etc.
+
+### Motion
+
+Motion text: Kate Herteck as Chair, Christina Koch as Vice-Chair, Sue McClatchy as Treasure and Rayna Harris as Secretary.
+Background: All persons volunteered for the postions. All positions had only one contestor.
+- Presented by: Jason
+- Result: Passed
+- Action item(s)/Assignee(s): Kate: Write blog post announcing the new officers.
+
+## Item 1 Executive Director's Update and Questions [10 min, until 0:25]
+
+- Hiring update - 7 Candidates have applied so-far
+- Fiscal sponsor update - workflows are improving with NumFOCUS and we haven’t found an outside fiscal sponsor that is a good fit for us yet.
+- Staff have been working to automate key recurring tasks, certificate creation, quarterly pre/post workshop survey reporting, instructor training and workshop logistics.
+- We’ve been approached by the Midwest Big Data Hub to be involved in some training efforts there. More to come soon.
+- Compute Canada is going to shift to a capacity building catalyzing role. They had been the source for instructors in Canada, but they don’t want to be the bottleneck anymore.
+
+#### Minutes
+
+- a lot of people interested in Carpentries in New Zealand
+- new partnerships,
+- planning interviews for the community coordinator position
+- workflows with numfocus are improving
+- Stuff works on automating workflows:
+- certificates based on data in AMY
+- plots on influence of the workshops
+- Midwest data hub is interested in collaboration with SWC
+- Compute Canada is not thinking of growing the instructor pool more than they already have.
+- They will shift to community building role.
+- ELIXIR all hands meet in Rome coming soon
+- Allegra Via will have a workshop there
+- CW17 in Leeds too:
+- Jonah will give a tutorial on building new lesson
+- Numfocus: we now see the reports on how much money we have, what’s paid what’s due
+- Survey from advisory committee with 30 respondents.
+- Question should SC be CCd on AC mail?
+
+ ## Item 2 Instructor Director's Update and Questions [5 min, until 0:30]
+
+#### Minutes
+- Erin is not present. No update.
+- Erin will not sit in for every SC meeting.
+- there is a discussion about if the trainer can also do a demo session for the person who he taught; more info: https://github.com/swcarpentry/instructor-training/issues/392
+- there is a need for more info on certification process
+- There is an agenda 3 months forward where new trainers can fill in the slots when they’re available 
+- **Action**: Christina will have a discussion with Erin formalizing processes regarding training and certification (with Belinda)
+- **Action**:Rayna & Christina will have report back from trainer subcommittee for each meeting rather than have a instructor director’s update.
+
```

Comment:
I think we also agreed on putting in a more formal document describing who should decide what re training, what can the committee decide on, what can Erin decide on, and what should be the purview of the SC(s).